src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs

```diff
@@ -36,11 +36,6 @@ public VacancyManageOrchestrator(DisplayVacancyViewModelMapper vacancyDisplayMap
 
         public async Task<ManageVacancy> GetVacancyDisplayViewModelAsync(Vacancy vacancy)
         {
-            if (vacancy.ClosingDate.HasValue && vacancy.ClosingDate.Value <= _timeProvider.Now)
```

Comment:
Should this be driven by state?

Extension Method?  vacancy.IsClosed(Datetime now)